instance/jmsml_D_Activity.xml

```diff
@@ -1132,6 +1132,9 @@
     </LegacySymbol>
     <LegacySymbol ID="INCIDENT_CRIMINAL_ACTIVITY_INCIDENT_CIVIL_RIOTING_SYM" Label="E*IPAC----*****" EntityID="INCIDENT" EntityTypeID="CRIMINAL_ACTIVITY_INCIDENT" EntitySubTypeID="CIVIL_RIOTING">
       <LegacyFunctionCode Name="2525C" DimensionOverride="I">AC----</LegacyFunctionCode>
+	  <LegacyEntity ID="INCIDENT_CRIMINAL_ACTIVITY_INCIDENT_CIVIL_RIOTING_LE" Label="EMS.INCDNT.CVDIS.CVRIOT" Graphic="E-I-AC----.svg">
```

Comment:
@Dbarnes1 - general comment.  All LegacyEntity structures should be indented.  They should line up with the LegacyFunctionCode above.